bentoml/_internal/cli/bento_management.py

```diff
@@ -52,16 +52,27 @@ def add_bento_management_commands(
     @click.option(
         "-o",
         "--output",
-        type=click.Choice(["tree", "json", "yaml", "path"]),
-        default="tree",
+        type=click.Choice(["json", "yaml", "path"]),
+        default="yaml",
     )
     def get(bento_tag, output):
         """Print Bento details by providing the bento_tag
 
         bentoml get FraudDetector:latest
         bentoml get FraudDetector:20210709_DE14C9
```

Comment:
Maybe add some output examples?

Reply:
I think output examples is a bit too long to be included here. But I made some improvements on the doc strings.